docs/_docs/plugins.md

```diff
@@ -810,6 +814,9 @@ LESS.js files during generation.
 
 #### Tags
 
+You can find a few useful plugins at the following locations:
+
+- [Jekyll-gist by Parkr](https://github.com/jekyll/jekyll-gist): Use the `gist` tag to easily embed a GitHub Gist onto your site. This works with public or secret gists.
```

Comment:
What Parrkr meant was simply to remove "by Parkr" from the link to `jekyll-gist`